### DISCUSSION TITLE
product token names

### DISCUSSION THREAD

**Discussion by jonlamb-gh** [keygen-sh/community#6] [Ideas]
:wave: 

Not sure how much merit this has, but I think it'd be handy if API tokens (specifically product-tokens) could be given a name.
I'm currently managing a list of product-token IDs and their owner or purpose by hand.
Mostly so I know which ones can be safely deleted later on.
If I could give them names, like "CI token for product uploads" or "Test token for website dev", I could get rid of my list.

**Comment by ezekg** [comment]
Hey @jonlamb-gh, this is a great idea. This shouldn't be too hard to add, so I'll get to it soon.

**Reply by ezekg** [reply]
Forgot to update you — as of Sep 2022, this is available. 👍

**Comment by flopp999** [comment]
I think this is solved
![image](https://github.com/keygen-sh/community/assets/21694965/1af3df94-88a2-489d-97a4-0fc9ce0a6ebb)
